public/index_symfony.php

```diff
@@ -38,6 +38,11 @@
     exit();
 }
 
+// Show all week plannings.
+if ($path == '/index' and !$date and $_SESSION['oups']['week']) {
+    header("Location: $base_url/week");
+}
+
```

Comment:
Ceci devrait être géré dans IndexController.php avec un return $this->redirectToRoute('xxx');
Je teste pour voir ce que ça donne

Reply:
corrigé par 272d7ca5b4fcd2d88ee41c5ba1b8c4d6e2a9cac9